htsget-http-actix/README.md

```diff
@@ -40,37 +42,45 @@ Since this service can be used in serverless environments, no `dotenv` configura
 | HTSGET_CREATED_AT | Date of the creation of the service. | "" |
 | HTSGET_UPDATED_AT | Date of the last update of the service. | "" |
 | HTSGET_ENVIRONMENT | Environment in which the service is running. | Testing |
+For more information about the regex options look in the [documentation of the regex crate](https://docs.rs/regex/).
 
 ## Example cURL requests
 
 As mentioned above, please keep in mind that the server will take the path where you executed it as base path to search for files. Here's a selection on what to ask this server for:
 
 ### GET
 
-```bash
+```shell
 $ curl '127.0.0.1:8080/variants/data/vcf/sample1-bcbio-cancer'
 ```
 
 ### POST
 
-```bash
+```shell
 $ curl --header "Content-Type: application/json" -d '{}' '127.0.0.1:8080/variants/data/vcf/sample1-bcbio-cancer'
 ```
 
 ### Parametrised GET
 
-```bash
+```shell
 $ curl '127.0.0.1:8080/variants/data/vcf/sample1-bcbio-cancer?format=VCF&class=header'
 ```
 
 ### Parametrised POST
 
-```bash
+```shell
 $ curl --header "Content-Type: application/json" -d '{"format": "VCF", "regions": [{"referenceName": "chrM"}]}' '127.0.0.1:8080/variants/data/vcf/sample1-bcbio-cancer'
 ```
 
 ### Service-info
 
-```bash
+```shell
 $ curl 127.0.0.1:8080/variants/service-info
 ```
+
+## Example Regular expressions
+In this example 'data/' is added after the first '/'.
+```shell
+$ HTSGET_REGEX='(?P<group1>.*?)/(?P<group2>.*)' HTSGET_REPLACEMENT='$group1/data/$group2' cargo run --release -p htsget-http-actix
```

Comment:
It's good to have the general idea on how to match with regexes, but I think that it could be more appropriate to have a particular example that targets particular files, as the reference impl does in:

https://github.com/ga4gh/htsget-refserver#configuration---reads-object

i.e:

```
                "sources": [
                    {
                        "pattern": "^tabulamuris\\.(?P<accession>10X.*)$",
                        "path": "https://s3.amazonaws.com/czbiohub-tabula-muris/10x_bam_files/{accession}_possorted_genome.bam"
                    },
                    {
                        "pattern": "^tabulamuris\\.(?P<accession>.*)$",
                        "path": "https://s3.amazonaws.com/czbiohub-tabula-muris/facs_bam_files/{accession}.mus.Aligned.out.sorted.bam"
                    }
```

Reply:
I have tried to come up with a better example, but I haven't find anything that satisfies me. Could you give me a hint on what exactly would be okay? My original intent was to show the use of capture groups

Reply:
If you visit https://s3.amazonaws.com/czbiohub-tabula-muris/, take a look at the example data files that sit there and imagine that you [only want to expose samples sequenced by 10X technology](https://www.10xgenomics.com/) as "sources" of data, that could be a good documentation example and it's very similar to what htsget-refserver from GA4GH does above :)

Reply:
Hmm, but we don't have that type of storage right now. Wouldn't an example like that be a little confusing?

Reply:
Ooh, you are totally right, we only have "local" and the "S3" storage backend that @mmalenic is working on right now. Ok, leave it as-is for now, fix the conflicts and merge, we'll rephrase/update when we get the appropriate (http?) storage backend.

Reply:
Okay!